languages/elixir/exercises/concept/conditionals/.docs/hints.md

```diff
@@ -0,0 +1,21 @@
+### General
+
+- The [atom type is described here][atom]
+
+### 1. Return the logging code label
+
+- You can use a [`cond/1` function][cond] to elegantly handle the various log codes.
+- You can use [equality operators][equality] to compare integers for strict type equality.
+
+### 2. Support unknown logging codes
+
+- There is a [way to specify a default branch][cond] in a cond function that can be used to catch unspecified cases.
+
+### 3. Send an alert
+
+- You can use a [`cond/1` function][cond] to compare the code's label, determine whether an alert should be sent.
```

Comment:
This line gives away the solution. Maybe you could rephrase a bit to not include the solution?

Reply:
I just shortened it to something more similar to the first hint.